components/search/index.css

```diff
@@ -298,6 +325,18 @@ governing permissions and limitations under the License.
 		var(--mod-search-bottom-to-text, var(--spectrum-search-bottom-to-text)) -
 			var(--mod-search-border-width, var(--spectrum-search-border-width))
 	);
+	padding-inline-start: calc(
+		var(--mod-search-edge-to-text, var(--spectrum-search-edge-to-text)) -
```

Comment:
Is there a reason for using `edge-to-text` rather than `edge-to-visual` here?

Reply:
Great catch, I missed this one.